extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @copyright Microsoft 2020
+ */
+
+import {
+    IPlugin, IConfiguration, IAppInsightsCore,
+    BaseTelemetryPlugin, CoreUtils, ITelemetryItem,
+    IProcessTelemetryContext, ITelemetryPluginChain,
+    _InternalMessageId, ICustomProperties, 
+    LoggingSeverity
+} from "@microsoft/applicationinsights-core-js";
+import { IConfig, Util } from "@microsoft/applicationinsights-common";
+import { 
+    IClickAnalyticsConfiguration, IContentHandler, 
+    IAutoCaptureHandler, IPageActionTelemetry 
+} from './Interfaces/Datamodel';
+import {
+    mergeConfig, BehaviorMapValidator,
+    BehaviorValueValidator, BehaviorEnumValidator, _ExtendedInternalMessageId
+    } from './common/Utils';
+import { PageAction } from './events/PageAction';
+import { AutoCaptureHandler } from "./handlers/AutoCaptureHandler";
+import { DomContentHandler } from "./handlers/DomContentHandler";
+export { BehaviorMapValidator, BehaviorValueValidator, BehaviorEnumValidator }
+
+export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
+    public identifier: string = 'ClickAnalyticsPlugin';
+    public priority: number = 181;
+    public version = '#version#';
```

Comment:
This will need to be the version number as 3P doesn't yet have automated version updating.